lib/origen_testers/smartest_based_tester/base.rb

```diff
@@ -70,6 +70,9 @@ class Base
       # format (SMT8 only)
       attr_accessor :zip_patterns
 
+      # Program comment character that affects what portions of a gnerated flow file to ignore during diff
+      attr_reader :program_comment_char
+
```

Comment:
Where does this actually get used?

Reply:
@ginty [here](https://github.com/Origen-SDK/origen_testers/blob/282e99230d408277b0358fc952aa983205e35b0b/lib/origen_testers/generator.rb#L213), I think